install/kubernetes/helm/istio/charts/sidecarInjectorWebhook/templates/clusterrolebinding.yaml

```diff
@@ -14,6 +13,9 @@ roleRef:
   name: istio-sidecar-injector-{{ .Release.Namespace }}
 subjects:
   - kind: ServiceAccount
+{{- if .Values.global.rbacEnabled }}
     name: istio-sidecar-injector-service-account
-    namespace: {{ .Release.Namespace }}
-{{- end }}
\ No newline at end of file
+{{- else }}
+    name: default
+{{- end }}
+    namespace: {{ .Release.Namespace }}
```

Comment:
nit, can you remove the character at the end?

Reply:
sure, thanks

Reply:
you mean add new line?

Reply:
![image](https://user-images.githubusercontent.com/4461983/41267317-a956ad2c-6e2d-11e8-9ef2-8e8959322188.png)

Remove the red part.

Reply:
ok, red part is adding new line. will add it in a sec.